impactlab_tools/os/config.py

```diff
@@ -0,0 +1,24 @@
+import sys
+import yaml
+
+def getConfigDictFromFile(filePath):
+    with open(filePath, 'r') as f:
+        confDict = yaml.load(f)
+    return confDict
+
+def getConfigDictFromSysArgv():
+    with open(sys.argv[1], 'r') as f:
+        confDict = yaml.load(f)
+
+        for key in serverConfig:
```

Comment:
Yes, `serverConfig` is the globally loaded configuration.

I think your other question is separate.  While one could do that, it means that the "idea" behind `getConfigDictFromFile` would be that it loads the server's main configuration file unless requested otherwise.  I would prefer `getConfigDictFromFile` to just load whatever file is requested, and for another function to determine the default at the point where it decides that it wants to load a configuration file and doesn't have any other information about which file to load.